packages/fuels/src/setup-launch-node-server.ts

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-console */
+import type { LaunchNodeOptions, LaunchNodeResult } from '@fuel-ts/account/test-utils';
+import { launchNode } from '@fuel-ts/account/test-utils';
+import http from 'http';
+import type { AddressInfo } from 'net';
+
+process.setMaxListeners(Infinity);
+
+async function parseBody(req: http.IncomingMessage) {
+  return new Promise<string>((resolve, reject) => {
+    const body: Buffer[] = [];
+    req.on('data', (chunk) => {
+      body.push(chunk);
+    });
+    req.on('end', () => {
+      resolve(JSON.parse(body.length === 0 ? '{}' : Buffer.concat(body).toString()));
```

Comment:
This should be in a try/catch
```suggestion
 try {
       resolve(JSON.parse(body.length === 0 ? '{}' : Buffer.concat(body).toString()));
    } catch (err) {
        reject(err);
    }
    ```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2378/commits/7a1ebbfe1a2944f5aebf3de9e75bd6003ba8b497